docs/guides/menu.json

```diff
@@ -0,0 +1,10 @@
+{
+  "title": "Guides",
+  "items": [
+    "hosting",
```

Comment:
The sorting of these categories don't make sense to me. Is it alphabetical? Or priority based?

Reply:
I can update to be alphabetical.